react-toggle-switch-component/dist/styles.css

```diff
@@ -0,0 +1,59 @@
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #e2e2e2;
+  border-radius: 20px;
+}
+
+.sliderClicked {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: green;
+  border-radius: 20px;
+}
+
+.slider::before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  transition: 0.4s;
+  border-radius: 20px;
+}
+
+.sliderClicked::before {
+  position: absolute;
+  content: "";
```

Comment:
I'll let you keep your `::before` and `content` if you can explain what they're doing here.